README.md

```diff
@@ -9,3 +9,25 @@ It detects whether the container environment contains secret references (`secret
 1. It prefixes the target containers command with `<path/to/volume>/secrethub run --`.
 
 This project is based on and heavily inspired by [Berglas's Kubernetes Mutating Webhook](https://github.com/GoogleCloudPlatform/berglas/tree/v0.5.1/examples/kubernetes).
+
+## Deploy the Webhook
+
+The simplest method to deploy the webhook is in a serverless function. Below we outline the steps to take to deploy the webhook to a Google Cloud Function.
+We're also [working on](https://github.com/secrethub/secrethub-kubernetes-mutating-webhook/pull/2) a way to deploy the webhook in the Kubernetes cluster itself.
+
+You can deploy the webhook to a Google cloud function using the following steps:
+
+1. Deploy the webhook to a Google Cloud Function:
+```sh
+gcloud functions deploy secrethub-mutating-webhook --runtime go113 --trigger-http
```

Comment:
This results in the following error for me:
```
ERROR: (gcloud.functions.deploy) OperationError: code=3, message=Build failed: # serverless_function_app
./main.go:24:43: cannot refer to unexported name webhook.secrethub
./main.go:24:43: undefined: webhook.secrethub
./main.go:24:66: undefined: mutating
./main.go:24:75: undefined: webhook
./main.go:26:44: cannot refer to unexported name webhook.secrethub
./main.go:26:44: undefined: webhook.secrethub
./main.go:26:67: undefined: mutating
./main.go:26:76: undefined: webhook; Error ID: 0eec69b1
```

Reply:
Ah, yes, the entrypoint is missing:

```suggestion
gcloud functions deploy secrethub-mutating-webhook --runtime go113 --entry-point F --trigger-http
```